lib/string.ts

```diff
@@ -6,3 +9,24 @@ export function cutMiddle(text = '', left = 6, right = 4): string {
   if (text.length <= left + right) return text;
   return `${text.substring(0, left)}...${text.substring(text.length - right, text.length)}`;
 }
+
+const builtInUnits = ['wei', 'kwei', 'mwei', 'gwei', 'szabo', 'finney', 'ether'];
+
+export function formatValue(value: BigNumber, type: string | number, dp = 2, withCommas = true): string {
```

Comment:
I added some comments about this function [here](https://app.shortcut.com/dux-makerdao/story/1025/create-helpers-for-parseether-parseunit-formatether-formatunit?vc_group_by=day&cf_workflow=500000005&ct_workflow=all)